bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/BoschSHCBridgeHandler.java

```diff
@@ -356,20 +356,15 @@ public void onComplete(@Nullable Result result) {
                                             if (BoschSHCHandler.class.isInstance(baseHandler)) {
                                                 BoschSHCHandler handler = (BoschSHCHandler) baseHandler;
 
-                                                if (handler != null) {
+                                                handled = true;
```

Comment:
Handler cannot be null here?

Reply:
My IDE suggested the removal because of "dead code" ;) The isInstance method already checks for null: https://stackoverflow.com/a/17923355/1346183